### ISSUE TITLE
Support for iOS Devices

### ISSUE THREAD

**Issue by t-scutulatus** [StollD/proton-webdav-bridge#1]
Could it be possible to deploy the bridge in an iPhone?

**Comment by StollD** [comment]
I don't think so, at least not without major changes. But I don't have experience with mobile (and especially iOS) development.
